prow/manifests/ingress.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  namespace: default
+  name: prow
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    # Change this to your issuer when using cert-manager. Does
+    # nothing when not using cert-manager.
+    # cert-manager.io/cluster-issuer: letsencrypt-prod
+spec:
+# Uncomment tls part only after you configure TLS
+  # tls:
+  # - hosts:
+  #   - prow.apps.test.metal3.io
+  #   secretName: metal3-io-tls
```

Comment:
TLS not set up yet?

Reply:
Good catch, forgot to uncomment it. In the current cluster TLS is enabled.

Reply:
I think I now remember why I commented out. Initially I applied Ingress without TLS, and added these lines only after I have installed cert-manager and applied cluster-issuer and I documented it in [here](https://github.com/Nordix/metal3-project-infra/tree/prow-update/feruz/prow#secure-ingress-with-cert-manager-and-lets-encrypt). Not sure if there would be any issue when we apply Ingress with TLS while there is no cert-manager & cluster-issuer yet on the cluster.

Reply:
I think that would be ok, everything would come up once cert-manager creates the certificates. I would try to enable it, to ensure it does not accidentally get forgotten

Reply:
Fixed, thanks.